### DISCUSSION TITLE
Best approach of moving a number of packages

### DISCUSSION THREAD

**Discussion by jeffreylanters** [openupm/openupm#2047] [Q&A]
Hi,

I'm considering to move all of the packages from https://github.com/elraccoone to my personal profile since I'm the only one in the team and I don't like having packages scattered around all over GitHub. But most of these have a decent amount of users and stars, I don't want to break their OpenUPM registers.

What would be the best approach of moving these?

Thanks in advance!

**Comment by favoyang** [comment]
Hi, if you don’t change the package name, then you can send a PR to update all related packages meta files’s repo URL. It won’t break anything.
